src/renderer/components/AircraftSection/TrackSelector.tsx

```diff
@@ -54,7 +54,21 @@ export const Track = styled(BaseTrack)`
 
   cursor: pointer;
   
-  border: ${props => props.isSelected ? `solid ${colors.title} 2px` : 'solid transparent 2px'};
+  border: solid 3px;
+  
+  border-color: ${colors.cardBackground};
+    border-left-color: ${props => props.isSelected ? `${colors.cardSelected}` : `${colors.cardBackground}`};
+  &:hover {
+    border-color: ${colors.cardBackgroundHover};
+      border-left-color: ${props => props.isSelected ? `${colors.cardSelected}` : `${colors.cardBackgroundHover}`};
+  }
+  
+  transition: border-color linear 200ms;
+  transition: border-top-color linear 200ms;
+  transition: border-right-color linear 200ms;
+  transition: border-bottom-color linear 200ms;
+  transition: border-left-color linear 200ms;
+
 `;
```

Comment:
How about this CSS trick:

```
  border: ${props => props.isSelected ? `solid transparent 2px` : 'solid transparent 2px'};
  :after {
    content:'';
    position: absolute;
    top:5%;
    left:-2px;
    width: 5px;
    height: 90%;
    background-color: ${props => props.isSelected ? `${colors.cardSelected}` : `transparent` } ;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
  }
```

Result:
![image](https://user-images.githubusercontent.com/1502124/105254568-2d5edd80-5b82-11eb-9ca5-6c7588c5144c.png)

Reply:
Thanks for the suggestion, I am going to check this out. I knew there is a css solution but I have zero experience, so highly appreciated!